common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -423,11 +413,7 @@ def setUp(self, *args, **kwargs):
         """
         super(RadioProblemTypeTest, self).setUp(*args, **kwargs)
         self.problem_page.a11y_audit.config.set_rules({
-            'ignore': [
-                'section',  # TODO: AC-491
-                'aria-valid-attr',  # TODO: AC-292
```

Comment:
Make sure that all referenced AC tickets are closed (if they are specific to capa).